### PR TITLE
Hide id_guest field if it is disabled in configuration 

### PR DESCRIPTION
**Pull request by aleeks**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Hide id_guest field if it is disabled in configuration + add boolean field into list template |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| How to test? | Enable and disable guest order & show cart page |

From : https://github.com/PrestaShop/PrestaShop/pull/6189

### CODE REVIEW

File: controllers/admin/AdminCartsController.php [hunk 1]
```diff
@@ -95,15 +95,18 @@ public function __construct()
...
+        );
+
+        if (Configuration::get('PS_GUEST_CHECKOUT_ENABLED')) {
+            $this->fields_list['id_guest'] = array(
                 'title' => $this->trans('Online', array(), 'Admin.Global'),
                 'align' => 'text-center',
                 'type' => 'bool',
                 'havingFilter' => true,
                 'class' => 'fixed-width-xs',
-                'icon' => array(0 => 'icon-', 1 => 'icon-user')
```
Comment (mickaelandrieu) [inline controllers/admin/AdminCartsController.php]: why did you remove `icon` parameter ?

**Review by mickaelandrieu — APPROVED** [review]

File: controllers/admin/AdminCartsController.php (hunk 1 above)
Comment (aleeks) [inline controllers/admin/AdminCartsController.php]: Because, they didn't work : 
mickael

And, in this case, **yes or no is better**

**Comment by mickaelandrieu** [comment]
Thank you @aleeks
